scripts/variantstore/wdl/GvsCreateFilterSet.wdl

```diff
@@ -138,53 +150,137 @@ workflow GvsCreateFilterSet {
+                    omni_resource_vcf = omni_resource_vcf,
+                    omni_resource_vcf_index = omni_resource_vcf_index,
+                    one_thousand_genomes_resource_vcf = one_thousand_genomes_resource_vcf,
+                    one_thousand_genomes_resource_vcf_index = one_thousand_genomes_resource_vcf_index,
+                    dbsnp_resource_vcf = dbsnp_resource_vcf,
+                    dbsnp_resource_vcf_index = dbsnp_resource_vcf_index,
+                    use_allele_specific_annotations = true,
+                    disk_size = small_disk
+            }
+        }
 
-        sites_only_variant_filtered_vcf = MergeVCFs.output_vcf,
-        sites_only_variant_filtered_vcf_index = MergeVCFs.output_vcf_index,
+        call Tasks.GatherTranches as SNPGatherTranches {
+            input:
+                tranches = SNPsVariantRecalibratorScattered.tranches,
+                output_filename = filter_set_name + ".snps.gathered.tranches",
+                mode = "SNP",
+                disk_size = small_disk,
+                gatk_override = gatk_override
+        }
 
-        snp_recal_file = SNPsVariantRecalibrator.recalibration,
-        snp_recal_file_index = SNPsVariantRecalibrator.recalibration_index,
-        snp_recal_tranches = SNPsVariantRecalibrator.tranches,
+        scatter (idx in range(length(ExtractFilterTask.output_vcf))) {
+            call CreateFilterSetFiles as CreateFilterSetFilesScattered {
+                input:
+                    gatk_override = gatk_override,
+                    filter_set_name = filter_set_name,
+                    output_directory = tmp_output_directory,
+                    sites_only_variant_filtered_vcf = MergeVCFs.output_vcf,
+                    sites_only_variant_filtered_vcf_index = MergeVCFs.output_vcf_index,
+                    snp_recal_file = SNPsVariantRecalibratorScattered.recalibration[idx],
+                    snp_recal_file_index = SNPsVariantRecalibratorScattered.recalibration_index[idx],
+                    snp_recal_tranches = SNPGatherTranches.tranches_file,
+                    indel_recal_file = IndelsVariantRecalibrator.recalibration,
+                    indel_recal_file_index = IndelsVariantRecalibrator.recalibration_index,
+                    indel_recal_tranches = IndelsVariantRecalibrator.tranches,
+                    index = idx,
+                    service_account_json = service_account_json,
+                    query_project = query_project
+            }
+        }
+    }
 
-        indel_recal_file = IndelsVariantRecalibrator.recalibration,
-        indel_recal_file_index = IndelsVariantRecalibrator.recalibration_index,
-        indel_recal_tranches = IndelsVariantRecalibrator.tranches,
+    if (GetNumSamples.num_samples <= snps_variant_recalibration_threshold) {
+        call SNPsVariantRecalibrator as SNPsVariantRecalibratorClassic {
+            input:
+                sites_only_variant_filtered_vcf = MergeVCFs.output_vcf,
```

Comment:
GitHub diff is confusing… but is this call the same as the "legacy" call we were making?  no parameter changes, etc right?

Reply:
right!